commons/src/main/java/org/dhis2/commons/featureconfig/model/Feature.kt

```diff
@@ -1,5 +1,6 @@
 package org.dhis2.commons.featureconfig.model
 
 enum class Feature(val description: String) {
-    ANDROAPP_2557("Local Analytics 2.5")
+    ANDROAPP_2557("Local Analytics 2.5"),
+    ANDROAPP_4049("Pie Chart Analytics")
```

Comment:
I think that the idea using ANDROAPP_2557 which is the Epic of the Local Analytics is to gather together all the features of analytics, enabling and disabling it all in a row, so I would use ANDROAPP_2557 as well in you case.

Reply:
Ah ok, I thought we have to add one feature for each ticket, I'll change it now